source/documentation/standards/code-freezes.html.md.erb

```diff
@@ -0,0 +1,133 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Avoid code freezes
+last_reviewed_on: 2023-06-19
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+Teams building systems for the MOJ should avoid code freezes whenever
+possible as they generally increase the risks to your service and those
+services around it.
+
+While code freezes are intended to reduce uncertainty and change when
+making changes to a system, they are typically a symptom of low trust
+in existing change processes around a given system, or of a previous
+traumatic event in the system's past.
+
+In most cases, it is better to make the system safer to change through
+[smaller, more regular changes](https://www.gov.uk/service-manual/technology/deploying-software-regularly)
+because [regular releases reduce risk](https://gds.blog.gov.uk/2012/11/02/regular-releases-reduce-risk/).
+
+## Code freezes introduce many risks
+
+There are various types of risk that code freezes introduce, both to
+the team's ability to resolve incidents and to the risk and impact
+profile of changes before, during, and after the code freeze.
+
+### Exceptional processes create uncertainty, and risk extending incidents
+
+In a team's day-to-day work, they will use a standard process for
+releasing changes to their systems. If they make changes regularly ([as
+they should](https://www.gov.uk/service-manual/technology/deploying-software-regularly)),
+they will be practiced experts in their normal processes. These
+processes will likely be well documented, well tested, and ideally
+primarily automated.
```

Comment:
🤔 remove "`ideally primarily` automated"

e.g. "These processes will likely be well documented, well tested, and automated."

Reply:
And easily reverted

Reply:
I actually think “easily reverted” is a symptom/outcome of the others. Like, if you can do a tested, automated release, you can revert that release as easily? What do you both think? Is it worth calling that out explicitly?

Reply:
I agree in part that it is an outcome of testing and automating but is its own benefit too: smaller more frequent releases allow for easier reversion than a big release with lots of features. Unpicking one specific aspect of a large release is a lot more difficult, even if it is tested and automated.

Reply:
Sorry to make it clearer, a lot of the reason why people favour change freezes is to prevent system breakage in time of minimal coverage, right? But if we practice small, frequent releases (with all the good stuff like testing and automating on top) even if a release fails, it's a lot easier to revert.